jekyll/_cci2/faq.md

```diff
@@ -230,24 +230,96 @@ CircleCI provides no guarantees about precision. A scheduled workflow will be ru
 
-### Is there a way to share plans across organizations and have them billed centrally?
+#### Is there a way to share plans across organizations and have them billed centrally?
 {:.no_toc}
 Yes, go to the Settings > Share & Transfer > Share Plan page of the CircleCI app to select the Orgs you want to add to your plan.
 
-### Can I set up billing for an organization, without binding it to my personal account?
+#### Can I set up billing for an organization, without binding it to my personal account?
 {:.no_toc}
 Yes, the billing is associated with the organization. You can buy while within that org's context from that org's settings page. But, you must have another GitHub Org Admin who will take over if you unfollow all projects. We are working on a better solution for this in a future update.
 
-### What is the definition of a container in the context of billing?
+#### What is the definition of a container in the context of billing?
 {:.no_toc}
 A container is a 2 CPU 4GB RAM machine that you pay for access to. Containers may be used for concurrent tasks (for example, running five different jobs) or for parallelism (for example, splitting one job across five different tasks, all running at the same time). Both examples would use five containers.
 
+---
+
+### Credit Usage Plans
+
+#### How do the new pricing plans affect me as a customer?
+{:.no_toc}
+For the vast majority of customers, you can keep your current plan for now and this simply represents a new option you may want to consider. 
+
+#### What are credits?
+{:.no_toc}
+Credits are used to pay for your usage based on machine type and size. Credits
+can also be used to pay for features, such as Docker Layer Caching.
+
+For example, the 25,000 credit package would provide 2,500 build minutes when
+using a single machine at the default rate of 10 credits per minute. The same package would last 1,250 minutes when using 2x parallelism or 250 minutes at 10x parallelism.
+
+#### If a container is used for under one minute, do I have to pay for a full minute? 
+{:.no_toc}
+You pay to the next nearest credit. First we round up to the nearest second, and then up to the nearest credit.
+
+#### How do I buy credits? Can I buy in any increments?
+{:.no_toc}
+Every month, you are charged for your selected credit package at the beginning of the month.
+
+#### How do I buy credits? Can I buy in any increments?
+{:.no_toc}
+Every month, you are charged for your selected credit package at the beginning of the month.
+
+#### What do I pay for?
+{:.no_toc}
+You can choose to pay for premium features per active user, compute, and optionally, premium support.
+
+- Access to features, such as new machine sizes, and Docker Layer Caching (DLS) are paid with a monthly fee of $15 per active user. 
```

Comment:
> Docker Layer Caching (DLC)

Reply:
DLC is paid for with credits per usage, similar to compute credits. It is not paid for via the $15 per active user charge.